### PR TITLE
Hook up import/totals endpoint to Historical Data Import screen

### PR DESCRIPTION
**Pull request by Aljullu**
Part of #1851.

Hooks up the `/wc/v4/reports/import/totals` endpoint with the Historical Data Import page.

### Screenshots
image

### Detailed test instructions:
- ~First you will need to merge the branch from #2189 (git merge add/1850-import-status).~ (no longer necessary)
- In `client/analytics/settings/historical-data/layout.js` line 70 set: `const showImportStatus = true;`, so progress bars are displayed.
- Go to _Settings_ and scroll down to _Import Historical Data_.
- Verify there are numbers next to the progress bars.
- Change the number of days to import and toggle the 'skip imported' checkbox and verify the numbers update accordingly.

### CODE REVIEW

File: client/analytics/settings/historical-data/progress.js [hunk 1]
```diff
@@ -3,20 +3,25 @@
...
+import { isNil } from 'lodash';
 
 function HistoricalDataProgress( { label, progress, total } ) {
+	let text = sprintf( __( 'Imported %(label)s', 'woocommerce-admin' ), {
+		label,
+	} );
+
+	if ( ! isNil( progress ) && ! isNil( total ) ) {
+		text +=
+			' ' +
```
Comment (jeffstieler) [inline client/analytics/settings/historical-data/progress.js]: I know this wasn't introduced in this PR, but concatenation like this breaks proper RTL translations.

**Review by jeffstieler — APPROVED** [review]
Code looks good and tests well - nice work @Aljullu!

**Comment by LevinMedia** [comment]
heya @Aljullu I set up a jurassic ninja site and populated it with 2500 orders and 500 customers. When I go to the settings screen, I'm seeing the progress bar, but I'm not getting any counts for the selected time period. 

Screen Shot 2019-05-15 at 3 28 36 PM

Cycling through the different time presets doesn't change anything either. When I changed the time period to custom the page refreshed and was more or less blank on reload:

Screen Shot 2019-05-15 at 3 29 42 PM

File: client/analytics/settings/historical-data/progress.js (hunk 1 above)
Comment (Aljullu) [reply client/analytics/settings/historical-data/progress.js]: Thanks for the review @jeffstieler! It should be fixed here:

a43186d6f832d686e65b65c6f336ad34a538bae1

**Comment by Aljullu** [comment]
@LevinMedia Did you merge the endpoints branch before testing it? (`git merge add/1850-import-status`) I guess that might be the cause of the issues.

This step is no longer necessary now, so maybe you could test it again pulling the last changes from this branch. I updated the test instructions in the PR description accordingly. If it's still not working, could you send me a link to the site so I can take a look? Thank you!

**Pull request merged**